logging/impl.go

```diff
@@ -98,14 +100,34 @@ func (imp *impl) Sync() error {
 	return multierr.Combine(errs...)
 }
 
-func (imp *impl) With(args ...interface{}) *zap.SugaredLogger {
-	return imp.AsZap().With(args...)
-}
-
 func (imp *impl) WithOptions(opts ...zap.Option) *zap.SugaredLogger {
 	return imp.AsZap().WithOptions(opts...)
 }
 
+func (imp *impl) WithFields(args ...interface{}) {
+	for keyIdx := 0; keyIdx < len(args); keyIdx += 2 {
```

Comment:
Was this copied from `zap` code? Mind leaving a permalink here if so?

Reply:
this was copied and slightly modified from `formatw()` in this file